app/contribution-hub/prize-tap/components/Content.tsx

```diff
@@ -289,109 +340,104 @@ const PrizeTapContent = () => {
+                RaffleStatus.ONGOING == selectedFilter ? "text-gray100" : ""
+              }`}
+              onClick={() => handleSelectFilter(RaffleStatus.ONGOING)}
+            >
+              ongoing
+            </div>
+            <div
+              className={`${
+                RaffleStatus.VERIFIED == selectedFilter ? "text-gray100" : ""
+              }`}
+              onClick={() => handleSelectFilter(RaffleStatus.VERIFIED)}
+            >
+              verified
+            </div>
+            <div
+              className={`${
+                RaffleStatus.REJECTED == selectedFilter ? "text-gray100" : ""
+              }`}
+              onClick={() => handleSelectFilter(RaffleStatus.REJECTED)}
+            >
+              rejected
+            </div>
+            <div
+              className={`${
+                RaffleStatus.FINISHED == selectedFilter ? "text-gray100" : ""
+              }`}
+              onClick={() => handleSelectFilter(RaffleStatus.FINISHED)}
+            >
+              finished
             </div>
           </div>
-          <div
-            className={`${Styles.refillToken} h-auto md:h-[78px] mt-4 flex w-full justify-between overflow-hidden items-center`}
-          >
-            <div className="flex flex-col sm:flex-row justify-between w-full items-center py-5 px-7 text-white">
-              <div className="flex items-center relative">
-                <div>
-                  <p className="text-[16px] font-semibold">Offer a New Prize</p>{" "}
-                  <p className="text-[14px] text-gray100">
-                    Here you can provide an NFT or Token for Prize Tap.
-                  </p>
-                </div>
-                <Icon
-                  className="absolute left-0 sm:right-[-45px] top-[-17px]  h-[150px] sm:h-[80px]"
-                  iconSrc="/assets/images/provider-dashboard/prize-bg.png"
-                />
-              </div>
-              <div
-                onClick={() => handleSelectNewOffer(true)}
-                className="flex mt-5 z-[10] sm:mt-0 items-center justify-center cursor-pointer border-2 border-white rounded-xl bg-[#0C0C17] w-[226px] h-[46px]"
-              >
-                + Provide a New Prize
+        </div>
+        <div
+          className={`${Styles.refillToken} h-auto md:h-[78px] mt-4 flex w-full justify-between overflow-hidden items-center`}
+        >
+          <div className="flex flex-col sm:flex-row justify-between w-full items-center py-5 px-7 text-white">
+            <div className="flex items-center relative">
+              <div>
+                <p className="text-[16px] font-semibold">Offer a New Prize</p>{" "}
+                <p className="text-[14px] text-gray100">
+                  Here you can provide an NFT or Token for Prize Tap.
+                </p>
               </div>
+              <Icon
+                className="absolute left-0 sm:right-[-45px] top-[-17px]  h-[150px] sm:h-[80px]"
```

Comment:
for centering in vertical position you may use these classes that aligns the item correctly based on the size

`top-1/2 -translate-y-1/2`